fake_switches/cmd/main.py

```diff
@@ -0,0 +1,42 @@
+import sys
+from twisted.internet import reactor
+from fake_switches.switch_configuration import SwitchConfiguration, Port
+from fake_switches.ssh_service import SwitchSshService
+from fake_switches.cisco.cisco_core import CiscoSwitchCore
+
+
+MODEL_MAPPING = {
+    'cisco_generic': CiscoSwitchCore
+}
+
+
+def get_switch_core(model_name):
+    core = MODEL_MAPPING[model_name]
+
+    return core(SwitchConfiguration("127.0.0.1", "my_switch",
+                                    ports=[Port("FastEthernet0/1"),
```

Comment:
Doing it now would close this file for modification and let it open for extension... further modifications would include having model specific derivatives of CiscoCore that overrides this function